src/blocked.c

```diff
@@ -108,9 +108,11 @@ void blockClient(client *c, int btype) {
 /* This function is called after a client has finished a blocking operation
  * in order to update the total command duration, log the command into
  * the Slow log if needed, and log the reply duration event if needed. */
-void updateStatsOnUnblock(client *c, long blocked_us, long reply_us){
+void updateStatsOnUnblock(client *c, long blocked_us, long reply_us, int errors){
```

Comment:
Maybe change the name? It looks like its the number of errors while actually its just indicating if an error occurred or not.